src/Card/CardExpandable.js

```diff
@@ -17,15 +17,27 @@ function getStyles() {
 
 class CardExpandable extends Component {
   static propTypes = {
+    closeIcon: PropTypes.node,
     expanded: PropTypes.bool,
     onExpanding: PropTypes.func.isRequired,
+    openIcon: PropTypes.node,
     style: PropTypes.object,
   };
 
   static contextTypes = {
     muiTheme: PropTypes.object.isRequired,
   };
 
+  getCorrectIcon(isExpanded, propIconOpen, propIconClose) {
+    if (isExpanded) {
+      if (propIconOpen) return propIconOpen;
+      return <OpenIcon />;
```

Comment:
I was referring to something like https://github.com/callemall/material-ui/blob/master/src/Tabs/Tabs.js#L80 but it might not work with react-docgen.
To see if that doesn't break the auto doc generation.